src/loader/mod.rs

```diff
@@ -378,6 +395,117 @@ impl KernelLoader for BzImage {
     }
 }
 
+#[cfg(feature = "arm64_pe")]
```

Comment:
nit: to shorten this, you could chain the `cfg`s: `#[cfg(all(feature = "arm64_pe", target_arch = "aarch64"))]

Reply:
(it would be even better if target-specific features were available :( )
BTW, nit-ish observation, I'd like to keep the target consistent with the feature name (i.e. call the feature `aarch64_pe` instead, or just plain `pe`), WDYT?

Reply:
Good idea to align all the names to `aarch64*`.